src/constants/types.ts

```diff
@@ -29,8 +29,13 @@ export interface ISeller {
     $numberDecimal: string;
   };
   coordinates: [number, number];
```

Comment:
Did you mean to remove this?

Reply:
> Did you mean to remove this?

Nevermind.  Can't we leverage the coordinates from the **sell_map_center** object instead?